aspnetcore/security/authentication/social/facebook-logins.md

```diff
@@ -16,43 +16,48 @@ uid: security/authentication/facebook-logins
 
 <a name=security-authentication-facebook-logins></a>
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT), [Pranav Rastogi](https://github.com/rustd), and [Valeriy Novytskyy](https://github.com/01binary)
+By [Rick Anderson](https://twitter.com/RickAndMSFT), [Pranav Rastogi](https://github.com/rustd), [Valeriy Novytskyy](https://github.com/01binary)
```

Comment:
Leave the conjunction "and" there. It took me like an hour to fix all of those across the docs. :smile: